Cargo.toml

```diff
@@ -41,7 +41,7 @@ core2 = { version = "0.3.0", optional = true, default-features = false }
 
 base64-compat = { version = "1.0.0", optional = true }
 bitcoinconsensus = { version = "0.19.0-3", optional = true }
-serde = { version = "1", features = [ "derive" ], optional = true }
+serde = { version = "1", default-features = false, features = [ "derive" ], optional = true }
 hashbrown = { version = "0.8", optional = true }
 
 [dev-dependencies]
```

Comment:
Since we removed `std` feature from `serde`, which breaks tests. Thus, in `dev-depencencies` we must add
```toml
serde = "1"
```

Reply:
Good catch. I will fix this.

Reply:
Fixed: 580650f

Reply:
Dropped this in favor of `serde-std` as discussed below.